index.html

```diff
@@ -225,10 +244,16 @@
 even prior to this change,
 its use was widespread in employment and other non-government interactions.
 
-
 ## Centralization of Trust
 
-TODO *exclusion... discouraging sites from working with the authorities they recognize... could lead to fragmentation along geographical lines ... one web principle ... biggest players...*
+Centralisation of trust can lead to a fragmented web, where access depends on which authorities
+a site or user is willing, or able, to work with.
+
+This risks excluding marginalized people. 
+For example, a visitor, migrant, or refugee may not be able to, or may not feel safe to, use credentials
+from their country of origin. If major platforms only recognize a narrow set of issuers, or only
+recognize issuers tied to a user’s default jurisdiction, such as the country where their device 
+was purchased, this undermines the global and open nature of the web.
```

Comment:
```suggestion
recognize issuers tied to a specific jurisdiction.
The choice of jurisdiction is not often something that a user can choose,
but instead one dictated by factors outside of their control.
This could undermine the global and open nature of the web.```